clients/tari-l1-signer.ts

```
@@ -0,0 +1,130 @@
+  id: number,
+): Promise<SignerReturnType<MethodName>> {
+  return new Promise<SignerReturnType<MethodName>>((resolve, reject) => {
+    const event_ref = (resp: MessageEvent<SignerResponse<MethodName>>) => {
+      if (resp.data.resultError) {
+        window.removeEventListener('message', event_ref)
+        reject(resp.data.resultError)
+      }
+      if (
+        resp &&
+        resp.data &&
+        resp.data.id &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)